common/views/components/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,61 @@
+import { FunctionComponent } from 'react';
+import {
+  useUserInfo,
+  withUserInfo,
+} from '@weco/identity/src/frontend/MyAccount/UserInfoContext';
+import { useRouter } from 'next/router';
+import { font } from '../../../utils/classnames';
+import Space from '../styled/Space';
+import Layout12 from '../Layout12/Layout12';
+import styled from 'styled-components';
+
+const SignInContainer = styled.div`
+  display: flex;
+`;
+
+const StyledSignIn = styled(Space).attrs({
+  as: 'div',
+  className: font('hnm', 5),
+})`
+  margin-left: auto;
+  background: ${props => props.theme.color('smoke')};
+`;
+
+const SignIn: FunctionComponent = () => {
+  const { user, isLoading } = useUserInfo();
+  const router = useRouter();
+  return router?.pathname !== '/item'
+    ? (!isLoading && (
+        <Layout12>
+          <SignInContainer>
+            <StyledSignIn
+              h={{
```

Comment:
You could add the `v` and `h` attributes to the styled component declaration to tidy up the markup a bit?